pymemcache/test/test_utils.py

```diff
@@ -12,6 +13,26 @@ def test_get_set():
     assert client.get(b"hello") == 12
 
 
+@pytest.mark.unit()
+def test_get_set_string():
+    client = MockMemcacheClient()
+    assert client.get("hello") is None
+
+    value = "world"
+    client.set("hello", value)
+    assert client.get("hello") == b"world"
+
+
+@pytest.mark.unit()
+def test_get_set_string_with_serde():
```

Comment:
This test was broken. It return a string with the real client, but mock implementation was returning bytes.